test/e2e_node/pods_container_manager_test.go

```diff
@@ -171,7 +171,7 @@ var _ = framework.KubeDescribe("Kubelet Cgroup Manager", func() {
 
 	Describe("Pod containers", func() {
 		Context("On scheduling a Guaranteed Pod", func() {
-			It("Pod containers should have been created under the cgroup-root", func() {
+			framework.ConformanceIt("Pod containers should have been created under the cgroup-root", func() {
```

Comment:
What is the purpose of this test? It looks like it's testing that the pod cgroup is created, not that the pod is actually running in that cgroup.

Reply:
To clarify my question, it feels like this is testing implementation rather than behavior. I'm not sure it makes sense as a conformance test from that perspective.